Classes/LinkHandler/GitHubLinkHandler.php

```diff
@@ -35,20 +34,22 @@ class GitHubLinkHandler implements LinkHandlerInterface
 
     protected ViewInterface $view;
 
-    protected PageRenderer $pageRenderer;
-
     protected array $configuration;
 
+    public function __construct(
+        // The page renderer is needed to register the JavaScript
+        private readonly PageRenderer $pageRenderer,
```

Comment:
I am not sure if this is correct as PageRenderer has state and is not a service AFAIK.

Reply:
The PageRenderer is a singleton therefore always the same instance will be injected by DI.

Reply:
Okay, found the injection of PageRenderer via constructor in core, so that is okay.